test/development.js

```diff
@@ -1,44 +0,0 @@
-// Packages
```

Comment:
I didn't add a test for this because the library is only to be used in production. Therefore, I don't find any benefit in checking for `NODE_ENV === 'development'`

The same applies for **production.js**.

the function `sendError` is updated accordingly